src/wovnio/html/HtmlConverter.php

```diff
@@ -74,6 +82,49 @@ private function insertSnippet($dom) {
 
     $token = $this->token;
     $insert_tags = array('head', 'body', 'html');
+    foreach($insert_tags as $tag_name) {
+      $parents = $dom->find($tag_name);
+      if (count($parents) > 0) {
+        $parent = $parents[0];
+        $parent->innertext = "<script src=\"//j.wovn.io/1\" data-wovnio=\"key=$token\" data-wovnio-type=\"backend_without_api\" async></script>" . $parent->innertext;
+        return;
+      }
+    }
+  }
+
+  /**
+   * Insert hreflang tags for all supported_langs
+   *
+   * @param simple_html_dom_node $dom
+   */
+  public function insertHreflangTags($dom) {
+    $lang_codes = $this->store->settings['supported_langs'];
+    foreach ($dom->find('link') as $node) {
+      $hreflangValue = $node->getAttribute('hreflang');
+      if (in_array(Lang::getCode($hreflangValue), $lang_codes)) {
+        // remove node
+        $node->outertext = '';
```

Comment:
I think that following code is more clear.
```
$node->parentNode->removeChild($node);
```

Reply:
There's no removeChild
http://simplehtmldom.sourceforge.net/manual_api.htm

Reply:
Okay.